tests/acceptance/features/apiGraph/getGroup.feature

```diff
@@ -52,3 +52,57 @@ Feature: get groups and their members
     When user "Brian" gets all the members of group "tea-lover" using the Graph API
     Then the HTTP status code should be "401"
     And the last response should be an unauthorized response
+
+
+    Scenario: admin user gets all groups along with their members information
+      Given user "Brian" has been created with default attributes and without skeleton files
+      And user "Carol" has been created with default attributes and without skeleton files
+      And group "tea-lover" has been created
+      And group "coffee-lover" has been created
+      And user "Alice" has been added to group "tea-lover"
+      And user "Brian" has been added to group "coffee-lover"
+      And user "Carol" has been added to group "tea-lover"
+      When user "Alice" retrieves all groups along with their members using Graph API
+      Then the HTTP status code should be "200"
+      And the group 'coffee-lover' should have the following member information
+        | displayName  | id        | mail              | onPremisesSamAccountName |
+        | Brian Murphy | %uuid_v4% | brian@example.org | Brian                    |
+      And the group 'tea-lover' should have the following member information
+        | displayName  | id        | mail              | onPremisesSamAccountName |
+        | Alice Hansen | %uuid_v4% | alice@example.org | Alice                    |
+        | Carol King   | %uuid_v4% | carol@example.org | Carol                    |
+
+
+  Scenario: normal user gets all groups along with their members information
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And group "tea-lover" has been created
+    And group "coffee-lover" has been created
+    And user "Alice" has been added to group "tea-lover"
+    And user "Brian" has been added to group "coffee-lover"
+    When user "Brian" retrieves all groups along with their members using Graph API
+    Then the HTTP status code should be "401"
+    And the last response should be an unauthorized response
+
+
+  Scenario: admin user gets a group along with their members information
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And group "tea-lover" has been created
+    And user "Alice" has been added to group "tea-lover"
+    And user "Brian" has been added to group "tea-lover"
+    When user "Alice" gets all the members information of group "tea-lover" using the Graph API
+    And the group 'tea-lover' should have the following member information
+      | displayName  | id        | mail              | onPremisesSamAccountName |
+      | Alice Hansen | %uuid_v4% | alice@example.org | Alice                    |
+      | Brian Murphy | %uuid_v4% | brian@example.org | Brian                    |
+
+  Scenario: normal user gets a group along with their members information
```

Comment:
@ScharfViktor i am in doubt whether this scenarios is supposed to work like this. As far as i remember, there is no mean s in `ocis` to hide `groups` info similar to that of `users`.

Reply:
I think it is bug. and I would expect 401
- when einstein calls all groups `graph/v1.0/groups/` - 401
- when einstein calls one group `graph/v1.0/groups/groupId` and one group with members `graph/v1.0/groups/groupId?$expand=members` - 200 with group and users info

Reply:
I so i will be creating an issue for it and may be discussed in the issue.

Reply:
issue created at https://github.com/owncloud/ocis/issues/5604